### ISSUE TITLE
Do you have any plans to release the frequency of the entity?

### ISSUE THREAD

**Issue by xiahaoyun** [princeton-nlp/EntityQuestions#9]
It is mentioned in the paper that “In our analysis, we use the Wikipedia hyperlink count as a proxy for an entity’s frequency.”.
My understanding is to traverse the entire Wikipedia to count the number of hyperlinks that link to an entity details page to represent the entity frequency.
This can be difficult, and implementations can introduce bias, Do you have any plans to release a file describing the frequencies of entities, or code that counts entity frequencies.

Thank you so much!

**Comment by a3616001** [comment]
Yes, we just counted the number of hyperlinks as the frequency.
Please find the entity frequency that we used [here](https://drive.google.com/file/d/1yv0dvFePGCiz8Us6oKWO4rOUxWvRpppg/view?usp=sharing).

**Comment by xiahaoyun** [comment]
Thanks for your prompt reply!
The file you provided was helpful.

**Comment by xiahaoyun** [comment]
I'm trying to extract entities from an existing query, looking for the frequency of entities. But I found that there are a large number of entities that can't find the corresponding frequency. Please is it because of something wrong with my code or is there really some entities missing from the file.

Below is my code:
https://gist.github.com/xiahaoyun/237534fc756da928cfbcae0e4f54f457

**Comment by a3616001** [comment]
We used the `label` of the Wikidata items to generate queries, which cannot be used to identify an entity; while in the entity-frequency file, the key is the corresponding Wikipedia `title` of the item, which is an unique ID for the entity. 

For example, the label of the Wikidata item [Q138518](https://www.wikidata.org/wiki/Q138518) is "**Princeton**", while the corresponding [Wikipedia title](https://en.wikipedia.org/wiki/Princeton,_New_Jersey) is "**Princeton, New Jersey**".

When building the dataset, we extracted (subject, relation, object) triples from [TREx](https://hadyelsahar.github.io/t-rex/) and used the `label` to generate queries, but we didn't record the corresponding item ID of each query. So, unfortunately, we are not able to map the entity back to ID and get its frequency.

Let me know if you have any further questions!
